consensus/exit_probability_randomizer/walker.c

```diff
@@ -112,8 +108,8 @@ static retcode_t random_walker_select_approver_tail(
       *has_approver_tail = false;
       return ret;
     }
-    HASH_FIND(hh, approvers_entry->approvers, approver->trits,
-              FLEX_TRIT_SIZE_243, approver_entry);
+    HASH_FIND(hh, approvers_entry->approvers, approver, FLEX_TRIT_SIZE_243,
+              approver_entry);
```

Comment:
Maybe replace this `HASH_FIND` by a function as well ?

Reply:
sure